reason_4.0/lib/core/minisite_templates/modules/events.php

```diff
@@ -4312,152 +4229,12 @@ function get_registration_forms_markup($event)
 	}
 
 	/**
-	 * Redirect to empty date if the given requested date is not a date the event occurrs
```

Comment:
I checked on this one since I couldn't remember either: this method was used by an earlier version of `get_registration_slots_markup()`, and is no longer used by any core code.